### DISCUSSION TITLE
How to show spoiler tags

### DISCUSSION THREAD

**Discussion by mrchief** [ArtiomTr/jest-coverage-report-action#96] [Q&A]
The readme says

> Showing spoiler in the comment for all new covered files. 🆕
> Showing spoiler in the comment for all files, in which coverage was reduced. 🔻

however, it is not clear how to enable this. As mentioned in https://github.com/ArtiomTr/jest-coverage-report-action/issues/91#issuecomment-869170871, I don't see spoiler tags getting created in the comment. 

That said, I think this is a fantastic action with the most comprehensive feature set, so I greatly appreciate all the work you're doing!

**Comment by ArtiomTr** [comment]
This action compares current branch coverage with target branch, so if you don't see any spoilers, try to add new function in your PR. Hope this solution will help you!

**Reply by mrchief** [reply]
Will do and report back how it goes. Thanks!
